tests/cli_test.go

```diff
@@ -563,8 +584,12 @@ func verifyExitCode(t *testing.T, expected, actual int) bool {
 	return success
 }
 
-func verifyOutput(t *testing.T, outputType, output string, patterns []MatchPattern) bool {
+func verifyOutput(t *testing.T, outputType, stdOutput string, patterns []MatchPattern) bool {
```

Comment:
`stdOutput` has a very specific meaning, and this has nothing to do with `stdout` or `stderr`

```suggestion
func verifyOutput(t *testing.T, outputType, rawOutput string, patterns []MatchPattern) bool {
```